src/core/api.c

```diff
@@ -796,6 +796,8 @@ MsQuicStreamStart(
     QUIC_STATUS Status;
     QUIC_STREAM* Stream;
     QUIC_CONNECTION* Connection;
+    BOOLEAN IsPriority = !!(Flags & QUIC_STREAM_START_FLAG_PRIORITY_WORK);
+    Flags &= ~QUIC_STREAM_START_FLAG_PRIORITY_WORK;
```

Comment:
Why are all these changes necessary?

Reply:
I found this is written for other api (StreamSend, SetParam etc.), so I added. Originally to check whether this fix stall issue